README.md

```diff
@@ -3,7 +3,7 @@
 ERDOS is a platform for developing self-driving cars and robotics applications.
 
 [![Crates.io][crates-badge]][crates-url]
-[![Build Status](https://travis-ci.org/erdos-project/erdos.svg)](https://travis-ci.org/erdos-project/erdos)
+[![Build Status](https://github.com/pschafhalter/erdos/workflows/CI/badge.svg)](https://travis-ci.org/erdos-project/erdos)
```

Comment:
Can you please fix the link?

Reply:
Oops, file didn't save before adding to git.